doc/deps.md

```diff
@@ -11,16 +11,13 @@ be installed from the official repository of the given Linux distribution.
 
 - **`git`**: For fetching and managing CodeCompass source code.
 - **`cmake`** and **`make`**: For building CodeCompass.
-- **`g++`**: For compiling CodeCompass. A version which supports C++14 features
-  is required. (Alternatively, you can compile with Clang.)
+-**`gcc-X`**: Version 7 or higher.
```

Comment:
Do we really need version 7 or higher? On Xenial ODB is built with version 5 even in the CI.

Reply:
Yeah I wanted to write 5 there.